ca/ca.go

```diff
@@ -311,18 +311,27 @@ func (ca *certificateAuthorityImpl) IssueCertificateForPrecertificate(ctx contex
 	if err != nil {
 		return nil, err
 	}
+
+	ca.log.AuditInfof("Signing cert: serial=[%s] regID=[%d] names=[%s] precert=[%s]",
+		serialHex, req.RegistrationID, strings.Join(issuanceReq.DNSNames, ", "), hex.EncodeToString(precert.Raw))
```

Comment:
Here we use `issuanceReq.DNSNames`. Below in the error and info case we use `precert.DNSNames`. I'm pretty sure those are equivalent but we should pick one and use it consistently. Also, since we log it in three places, we should probably do the `strings.Join` once and use the resulting variable in all three places.